koji/playbooks/users.yml

```diff
@@ -11,13 +11,17 @@
         name: "{{ user.name }}"
         state: "enabled"
         permissions: "{{ user.permissions }}"
+        koji: 'koji'
```

Comment:
Isn't this default?

Reply:
Nope. https://github.com/ktdreyer/koji-ansible#koji-profiles -- check the first line.